docs/mutations.md

```diff
@@ -37,133 +38,233 @@ any outstanding TODOs before opening issues.
 
 ---
 
-```{eval-rst}
-
-.. todo:: 
-    This has been copied directly from the api.rst file. Probably
-    needs some refactoring to use the newer docs arrangements.
-    Also all the labels need updating.
-```
+{func}`.sim_mutations` adds mutations to a tree sequence simulated with
+{func}`.sim_ancestry`. We can specify a uniform mutation rate or a
+{class}`.RateMap`, whether or not mutations should only occur at 
+{ref}`discrete coordinates <sec_mutations_discrete>`,
+a {ref}`specific time span within which to add mutations <sec_mutations_time_span>`,
+as well as how to handle any
+{ref}`existing mutations in a tree sequence <sec_mutations_existing>`.
 
-When running coalescent simulations it's usually most convenient to use the
-`mutation_rate` argument to the {func}`.simulate` function to throw neutral
-mutations down on the trees. However, sometimes we wish to throw mutations
-down on an existing tree sequence: for example, if we want to see the outcome
-of different random mutational processes on top of a single simulated topology,
-or if we have obtained the tree sequence from another program and wish to
-overlay neutral mutations on this tree sequence.
+One of the most powerful features of `msprime` is the flexibility of specifying
+the model under which mutations are simulated. Select from among
+the many predefined {ref}`models <sec_mutations_models>`, such as the 
+{ref}`infinite alleles model <sec_mutations_mutation_infinite_alleles>`, or
+{ref}`define your own <sec_mutations_matrix_mutation_models_details>`.
 
 
 (sec_mutations_rate)=
 
 ## Specifying rates
 
-```{eval-rst}
-.. todo:: Documentation on the ``rate`` parameter with examples.
+The `rate` parameter of {func}`.sim_mutations` determines the rate of mutations per unit
+of sequence length per generation. A rate must be specified for mutations to be generated.
 
+```{code-cell}
+ts = msprime.sim_ancestry(2, random_seed=1)
+mts = msprime.sim_mutations(ts, rate=1, random_seed=1)
+mts.num_mutations
 ```
 
+Running {func}`.sim_mutations` on this simulated tree of length 1 with `rate=1`
+produces 7 mutations.
+
+Specifying higher rates lead to proportionately more mutations:
+
+```{code-cell}
+ts = msprime.sim_ancestry(2, random_seed=1)
+mts = msprime.sim_mutations(ts, rate=5, random_seed=1)
+mts.num_mutations
+```
+
+It's also possible to provide a {class}`.RateMap` which specifies variable
+mutation rates over different stretches of the sequence. 
+
+```{code-cell}
+ts = msprime.sim_ancestry(10, sequence_length=7, random_seed=1)
+pos = [0, 2, 5, 7]
+rate = [1, 0, 0.5]
+ratemap = msprime.RateMap(pos, rate)
+mts = msprime.sim_mutations(ts, rate=ratemap, random_seed=1)
+mts.tables.sites.position
+```
+
+In this example, the mutation rate between positions 0 and 2 is 1,
+between 2 and 5 is 0, and between 5 and 10 is 0.5.
+
+Be sure that the final position of the {class}`.RateMap` is the same as the
+`sequence_length` of the tree sequence you're adding mutations to!
+
```

Comment:
One thing we're missing here is recurrent mutations; as you increase the rate, the probability you get multiple mutations at a site increases. Yan's new ts visualisation will help illustrate this. (We can add in the SVG and describe what you would see if you had the git version of tskit installed. The built docs will catch up soon enough)